autowelcome.py

```diff
@@ -0,0 +1,42 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+
+
+class Welcome(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.welcomes = self.open_storage('autowelcomes')
+
+    @cmd
+    def welcome(self, event):
+        """Create an autowelcome message for yourself."""
+        args = event.args
+
+        if len(args) < 1:
+            return "No new autowelcome message specified."
+
+        nick = event.meta['nick']
+        welcome = ' '.join(args[0:])
+        self.welcomes[nick] = welcome
+
+        return "Your new autowelcome is: {0}".format(welcome)
+
+    @cmd
+    def my_welcome(self, event):
+        """Shows users welcome message."""
+        nick = event.meta['nick']
+        if nick not in self.welcomes:
+            return "You don't have an autowelcome yet."
+        return self.welcomes[nick]
+
+    @cmd
+    def show_welcome(self, event):
+        """Show welcome message of a given user."""
+        args = event.args
+
+        if len(args) < 1:
+            return "No user specified."
+        else:
+            nick = args[0]
+
+        if nick not in self.welcomes:
+            return "User doesn't have welcome message yet."
```

Comment:
@JakubNvk it might be a good idea to include the user's nick in this message, especially if you would like to merge `my_welcome` with this function.